app/views/shared/_nav_bar.html.erb

```diff
@@ -164,7 +164,7 @@
 
 </nav>
 
-<div class="mobile-header">
+<div class="mobile-header d-print-none">
```

Comment:
Does this class simply hide this section in the printable version?

Reply:
Yes, exactly.